package.json

```diff
@@ -1,15 +1,12 @@
 {
   "name": "@ledgerhq/wallet-common",
   "version": "0.13.0",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "license": "MIT",
   "scripts": {
-    "preinstall": "node -e \"if (process.env.npm_execpath.indexOf('yarn') === -1) { console.log('\u001b[31mPlease use yarn\u001b[0m'); process.exit(1); }\"",
     "flow-typed": "flow-typed install -s",
-    "build": "yarn clean && yarn documentation && babel --source-maps -d lib src && flow-copy-source src lib",
-    "watch": "babel --watch --source-maps -d lib src && flow-copy-source -w -v src lib",
-    "prepare": "yarn flow-typed && yarn build",
```

Comment:
I think prepare was important, because it is called when doing a publish & you really want to build before publish.

that said, maybe we can have our own publish script and forbid us to use directly the `npm publish` / `yarn publish`

Reply:
The problem with having a prepare script is we no longer benefit of the yarn integrity check when we do `yarn`: everything is always rebuilt,  no matter something has changed. As you said, we may have our own publish script if we really want a "all-in-one" build and publish.

Reply:
yeah it's actually weird `yarn` would call `prepare` 😅 any other hook to override the "publish"? "prepublish" ?